interceptors/Mementifier.cfc

```diff
@@ -121,9 +121,42 @@ component{
 				thisName = ( md.keyExists( "entityName" ) ? md.entityName : listLast( md.name, "." ) );
 			}
 
-			thisMemento.defaultIncludes = ormGetSessionFactory()
-				.getClassMetaData( thisName )
-				.getPropertyNames();
+			var ORMService = new cborm.models.BaseORMService();
```

Comment:
The only thing I am seeing on this now is that this will break existing functionality, because now the orm auto includes will NOT include relationships!

Reply:
Yes, but that was a design flaw in the original implementation.  The previous code introduced a race condition when bi-directional relationships were present.  It expanded in to infinity.  If someone was using that functionality, they were playing with fire.

Reply:
Your existing implementations should be fine since you "mementified" before the ORM auto-defaultIncludes was added. :)